commons/active/banking/banking_attempt.avsc

```diff
@@ -0,0 +1,30 @@
+{
+  "namespace": "org.radarcns.active.banking",
+  "type": "record",
+  "name": "BankingAttempt",
+  "doc": "A Banking App Attempt to complete a transaction, which can contain a PIN, an Amount and a Confirm/Cancel Transaction event. It can also time out or end.",
+  "fields": [
+    { "name": "uid", "type": "string", "doc": "Unique ID for an Attempt."},
+    { "name": "time", "type": "double", "doc": "Time since the Unix Epoch (seconds)." },
+    { "name": "timeCompleted", "type": "double", "doc": "Time since the Unix Epoch (seconds)." },
+    { "name": "duration", "type": "double", "doc": "Duration of the activity (seconds)."},
+    { "name": "isComplete", "type":"boolean", "doc": "Describes if the Attempt was completed or not."},
+    { "name": "result", 
```

Comment:
Perhaps add a
```json
"name": "isCorrect",
"type": ["null", "boolean"],
"doc": "True if both the PIN and amount were correctly specified, false otherwise. Null if the PIN or amount have not been input, due to an early exit or timeout.",
"default": null
```

Reply:
In TransactionResult these are included like: 
CORRECT_CONFIRMED CORRECT_CANCELLED (both pin and amount correct)
WRONG_CONFIRMED, WRONG_CANCELLED (one of pin or amount was incorrect)

Do you want that differently put?